packages/invoices/src/OutboundShipment/DetailView/SidePanel.tsx

```diff
@@ -106,6 +111,69 @@ const RelatedDocumentsSection: FC = () => {
   );
 };
 
+export const PricingSection = () => {
+  const t = useTranslation('distribution');
+  const serviceLineModal = useToggle(false);
+  const { c } = useCurrency();
+  const { pricing } = useOutboundFields(['pricing']);
+
+  return (
+    <DetailPanelSection title={t('heading.charges')}>
+      <Grid container gap={0.5}>
+        <OutboundServiceLineEdit
+          isOpen={serviceLineModal.isOn}
+          onClose={serviceLineModal.toggleOff}
+        />
+
+        <PanelRow>
+          <PanelLabel fontWeight="bold">
+            {t('heading.stock-charges')}
+          </PanelLabel>
+        </PanelRow>
+        <PanelRow>
+          <PanelLabel>{t('heading.sub-total')}</PanelLabel>
+          <PanelField>{c(pricing.stockTotalBeforeTax).format()}</PanelField>
+        </PanelRow>
+        <PanelRow>
+          <PanelLabel>{t('heading.total')}</PanelLabel>
+          <PanelField>{c(pricing.stockTotalAfterTax).format()}</PanelField>
+        </PanelRow>
+        <PanelRow>
+          <PanelLabel fontWeight="bold">
+            {t('heading.service-charges')}
+          </PanelLabel>
+          <PanelField>
+            <IconButton
+              icon={<EditIcon />}
+              label={t('label.edit')}
+              onClick={serviceLineModal.toggleOn}
+            />
+          </PanelField>
+        </PanelRow>
+        <PanelRow>
+          <PanelLabel>{t('heading.sub-total')}</PanelLabel>
+          <PanelField>{c(pricing.serviceTotalBeforeTax).format()}</PanelField>
+        </PanelRow>
+        <PanelRow>
+          <PanelLabel>{t('heading.total')}</PanelLabel>
+          <PanelField>{c(pricing.serviceTotalAfterTax).format()}</PanelField>
+        </PanelRow>
+        <PanelRow>
+          <PanelLabel fontWeight="bold">{t('heading.totals')}</PanelLabel>
+        </PanelRow>
+        <PanelRow>
+          <PanelLabel>{t('heading.sub-total')}</PanelLabel>
+          <PanelField>{c(pricing.totalBeforeTax).format()}</PanelField>
+        </PanelRow>
+        <PanelRow>
+          <PanelLabel>{t('heading.total')}</PanelLabel>
+          <PanelField>{c(pricing.totalAfterTax).format()}</PanelField>
+        </PanelRow>
+      </Grid>
+    </DetailPanelSection>
```

Comment:
I should probably add `tax` here somewhere also? For each section? 🤷  Would need to calculate from the subtotals and totals 🤔

Reply:
yes, probably should. maybe not right now though?